mapwms.c

```diff
@@ -1910,6 +1887,22 @@ this request. Check wms/ows_enable_request settings.",
 
   }
 
+  /*
+  ** Apply vendor-specific filter if specified
+  */
+  if (filter) {
+    if (sld_url || sld_body) {
+      msSetError(MS_WMSERR,
+                 "Vendor-specific FILTER parameter cannot be used with SLD or SLD_BODY.",
+                 "msWMSLoadGetMapParams()");
+      return msWMSException(map, nVersion, NULL, wms_exception_format);
+    }
+
+    if (msWMSApplyFilter(map, nVersion, filter, need_axis_swap, wms_exception_format) == MS_FAILURE) {
+      return MS_FAILURE;/* msWMSException(map, nVersion, "InvalidFilterRequest"); */
```

Comment:
Is there a reason not to throw a msWMSException (I see this is from the original function).

Reply:
This is probably a better question for Dan. My solution was simply to move the query/filter code below style-handling block.

Reply:
That's probably a question for Dan.

Reply:
If it is a simple MS_FAILURE then I thins servers just return a HTTP 500 error which are hard to debug compared to an error message? Maybe one for @dmorissette but not a blocker.